logic/vote.js

```diff
@@ -194,42 +194,19 @@ Vote.prototype.getBytes = function (transaction) {
 };
 
 /**
- * Calls checkConfirmedDelegates based on transaction data and
- * merges account to sender address with votes as delegates.
- * @implements {checkConfirmedDelegates}
- * @implements {scope.account.merge}
- * @implements {slots.calcRound}
+ * Calls checkConfirmedDelegates based on transaction data
  * @param {transaction} transaction
  * @param {block} block
  * @param {account} sender
  * @param {function} cb - Callback function
  * @todo delete unnecessary var parent = this
  */
 Vote.prototype.apply = function (transaction, block, sender, cb) {
-	var parent = this;
-
-	async.series([
-		function (seriesCb) {
-			self.checkConfirmedDelegates(transaction, seriesCb);
-		},
-		function (seriesCb) {
-			parent.scope.account.merge(sender.address, {
-				delegates: transaction.asset.votes,
-				blockId: block.id,
-				round: slots.calcRound(block.height)
-			}, function (err) {
-				return setImmediate(cb, err);
-			});
-		}
-	], cb);
+	return setImmediate(cb);
 };
 
 /**
- * Calls Diff.reverse to change asset.votes signs and merges account to 
- * sender address with inverted votes as delegates.
- * @implements {Diff}
- * @implements {scope.account.merge}
- * @implements {slots.calcRound}
+ * Calls Diff.reverse to change asset.votes signs
```

Comment:
Descriptions are no longer valid - that can be addressed in separate issue.

Reply:
These are slated to be removed in a subsequent PR